src/client/src/sync/mod.rs

```diff
@@ -1,164 +1,159 @@
 mod controller;
 mod store;
-
 pub use store::*;
-pub use context::*;
-
-mod context {
-
-    use std::sync::Arc;
-    use std::fmt::Display;
-    use std::io::Error as IoError;
-
-    use tracing::{debug, trace, instrument};
-    use async_rwlock::RwLockReadGuard;
-    use once_cell::sync::Lazy;
 
-    use crate::FluvioError;
-    use crate::metadata::core::Spec;
-    use crate::metadata::store::LocalStore;
-    use crate::metadata::store::DualEpochMap;
-    use crate::metadata::store::MetadataStoreObject;
-    use crate::metadata::spu::SpuSpec;
-    use crate::metadata::core::MetadataItem;
-
-    pub(crate) type CacheMetadataStoreObject<S> = MetadataStoreObject<S, AlwaysNewContext>;
-
-    /// Timeout
-    static MAX_WAIT_TIME: Lazy<u64> = Lazy::new(|| {
-        use std::env;
-        let var_value = env::var("FLV_METADATA_TIMEOUT").unwrap_or_default();
-        let wait_time: u64 = var_value.parse().unwrap_or(60000); // up to 60 seconds
-        wait_time
-    });
+use std::sync::Arc;
+use std::fmt::Display;
+use std::io::Error as IoError;
+
+use tracing::{debug, trace, instrument};
+use async_rwlock::RwLockReadGuard;
+use once_cell::sync::Lazy;
+
+use crate::FluvioError;
+use crate::metadata::core::Spec;
+use crate::metadata::store::LocalStore;
+use crate::metadata::store::DualEpochMap;
+use crate::metadata::store::MetadataStoreObject;
+use crate::metadata::spu::SpuSpec;
+use crate::metadata::core::MetadataItem;
+
+pub(crate) type CacheMetadataStoreObject<S> = MetadataStoreObject<S, AlwaysNewContext>;
```

Comment:
Can you minimize change here so it's to easier to see diff? If more make it cleaner, can you do that in separate PR?

Reply:
I remember removing an inline module, I'll go back and undo that to reduce noise in the diff.